### PR TITLE
refact: Remove useless else

### PR DESCRIPTION
**Pull request by EwertonBello**

### CODE REVIEW

**Review by leandromoreira — CHANGES_REQUESTED** [review]
I think this changes were tackled at a previous PR.

File: m3u8/model.py [hunk 1]
```diff
@@ -817,7 +817,7 @@ def __init__(self, uri, stream_info, media, base_uri):
         self.base_uri = base_uri
 
         resolution = stream_info.get('resolution')
-        if resolution != None:
+        if resolution is None:
```
Comment (rokam) [inline m3u8/model.py:820]: Here you are changing the logic.

Comment (leandromoreira) [reply m3u8/model.py:820]: good catch

**Comment by EwertonBello** [comment]
> I think this changes were tackled at a previous PR.

Thanks, the commit was incorrect, it's fixed now!

File: m3u8/model.py [hunk 2]
```diff
@@ -817,7 +817,7 @@ def __init__(self, uri, stream_info, media, base_uri):
         self.base_uri = base_uri
 
         resolution = stream_info.get('resolution')
-        if resolution is None:
+        if resolution != None:
```
Comment (leandromoreira) [inline m3u8/model.py:820]: it should be `resolution is not None`? also isn't this related to your other open PR, I think you must split the code changes.

File: tests/test_model.py [hunk 3]
```diff
@@ -207,8 +207,8 @@ def test_segment_envivio_scte35_attribute():
 
 def test_segment_unknown_scte35_attribute():
     obj = m3u8.M3U8(playlists.CUE_OUT_INVALID_PLAYLIST)
-    assert obj.segments[0].scte35 is None
-    assert obj.segments[0].scte35_duration is None
+    assert obj.segments[0].scte35 == None
+    assert obj.segments[0].scte35_duration == None
```
Comment (leandromoreira) [inline tests/test_model.py:211]: same here, I think you're mixing your PRs

Comment (EwertonBello) [reply tests/test_model.py:211]: I believe it is the first commit, it has already been fixed in the second commit

File: m3u8/parser.py [hunk 4]
```diff
@@ -412,8 +412,7 @@ def _cueout_envivio(line, state):
     res = re.match('.*DURATION=(.*),.*,CUE="(.*)"', value)
     if res:
         return (res.group(2), res.group(1))
-    else:
-        return None
+    return None
```
Comment (davemevans) [inline m3u8/parser.py:415]: `None` will be returned implicitly, so this statement is only useful if it's necessary to be explicit for some reason

Comment (mauricioabreu) [reply m3u8/parser.py:415]: I think this case is one of those "explicit is better than implicit"

**Review by mauricioabreu — APPROVED** [review]

**Comment by mauricioabreu** [comment]
@EwertonBello can you rebase your branch against master? It has some conflicts. After rebasing I'll merge it.

**Comment by nedmax** [comment]
Opting for explicit

**Pull request closed without merging**